sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_configure.py

```diff
@@ -0,0 +1,210 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License in the project root for
+# license information.
+# --------------------------------------------------------------------------
+from logging import getLogger
+from typing import Any, Dict
+
+from azure.monitor.opentelemetry._constants import (
+    DISABLE_LOGGING_ARG,
+    DISABLE_METRICS_ARG,
+    DISABLE_TRACING_ARG,
+    EXCLUDE_INSTRUMENTATIONS_ARG,
+    INSTRUMENTATION_CONFIG_ARG,
+    LOGGER_NAME_ARG,
+    LOGGING_EXPORT_INTERVAL_MS_ARG,
+    LOGGING_LEVEL_ARG,
+    METRIC_READERS_ARG,
+    RESOURCE_ARG,
+    SAMPLING_RATIO_ARG,
+    TRACING_EXPORT_INTERVAL_MS_ARG,
+    VIEWS_ARG,
+)
+from azure.monitor.opentelemetry._types import ConfigurationValue
+from azure.monitor.opentelemetry.exporter import (
+    ApplicationInsightsSampler,
+    AzureMonitorLogExporter,
+    AzureMonitorMetricExporter,
+    AzureMonitorTraceExporter,
+)
+from azure.monitor.opentelemetry.util.configurations import _get_configurations
+from opentelemetry._logs import get_logger_provider, set_logger_provider
+from opentelemetry.instrumentation.dependencies import (
+    get_dist_dependency_conflicts,
+)
+from opentelemetry.instrumentation.instrumentor import BaseInstrumentor
+from opentelemetry.metrics import set_meter_provider
+from opentelemetry.sdk._logs import LoggerProvider, LoggingHandler
+from opentelemetry.sdk._logs.export import BatchLogRecordProcessor
+from opentelemetry.sdk.metrics import MeterProvider
+from opentelemetry.sdk.metrics.export import PeriodicExportingMetricReader
+from opentelemetry.sdk.resources import Resource
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
+from opentelemetry.trace import get_tracer_provider, set_tracer_provider
+from pkg_resources import iter_entry_points
+
+_logger = getLogger(__name__)
+
+
+_SUPPORTED_INSTRUMENTED_LIBRARIES = (
+    "django",
+    "fastapi",
+    "flask",
+    "psycopg2",
+    "requests",
+    "urllib",
+    "urllib3",
+)
+
+
+InstrumentationConfig = Dict[str, Dict[str, Any]]
+
+
+def configure_azure_monitor(**kwargs) -> None:
+    """
+    This function works as a configuration layer that allows the
+    end user to configure OpenTelemetry and Azure monitor components. The
+    configuration can be done via arguments passed to this function.
+    :keyword str connection_string: Connection string for your Application Insights resource.
+    :keyword Sequence[str] exclude_instrumentations: Specifies instrumentations you want to disable.
+    :keyword Resource resource: Specified the OpenTelemetry [resource][opentelemetry_spec_resource] associated with your application.
+    :keyword bool disable_logging: If set to `True`, disables collection and export of logging telemetry. Defaults to `False`.
```

Comment:
We think that it's important to have an intuitive way to disable logging, especially given that the other pillars have disable_<pillar> arguments. Furthermore, it makes sense for logger_name to default to "" and NOT disable the column as [default OTel behavior is to instrument the base logger](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-sdk/src/opentelemetry/sdk/_configuration/__init__.py#L243)

Reply:
@annatisch

Reply:
The other `disable` parameter features don't seem to impact the default Python environment in the same way

Reply:
Could you explain more what you mean by "impact the default Python environment" as well as what you are proposing?